### ISSUE TITLE
Add a standalone possibility

### ISSUE THREAD

**Issue by pvi-github** [cbeuw/Cloak-android#29]
Hello,

Cloak works on my laptop and it's a brilliant piece of software...

There are needs for a standalone option either for openvpn or for wireguard on Android.

I'd personally love to be able to use cloak on my mobile exactly the same way as i'm doing on the computer...

How about adding a tiny user interface allowing (advanced ?) users to run cloak on android without shadowsocks ?

Just to pass udp and tcp trafic just like cloak on *nix, mac and windows...

Kind regards,

**Comment by maelstrom256** [comment]
Agree. Cloak isn't a plugin, it's standalone port-tunneling service, it works without any additional software. It will be nice to have it as standalone tool also for android

**Comment by temperance447** [comment]
yes, like stunnel

**Comment by pvi-github** [comment]
Hello, project maintainters what do you think ?

**Comment by MarvinFS** [comment]
I agree and it seems like there are no option in the Internet now to run it on android as stand alone? (I use with WG as 
ck-client -u -c all-cloak.json -s 8x.xx.x6.xx -p 443 -l 1984
